packages/traceloop-sdk/traceloop/sdk/utils/package_check.py

```diff
@@ -1,20 +1,14 @@
-from importlib.metadata import distributions
+from importlib.metadata import Distribution, distributions
 
 
-def _get_package_name(dist):
-    # Try both 'Name' and 'name' keys to handle different metadata formats
-    if hasattr(dist, 'metadata') and dist.metadata is not None:
-        for key in ('Name', 'name'):
-            try:
-                return dist.metadata[key].lower()
-            except (KeyError, AttributeError):
-                continue
+def _get_package_name(dist: Distribution) -> str | None:
+    try:
+        return dist.name.lower()
```

Comment:
shouldn't be:

```suggestion
        return dist.name().lower()
```

?

Reply:
nope, it's a property
```
@property
def name(self):
    """Return the 'Name' metadata for the distribution package."""
    return self.metadata['Name']
```